zng/vector.go

```diff
@@ -39,22 +40,47 @@ func (t *TypeVector) Parse(in []byte) (zcode.Bytes, error) {
 	panic("zeek.TypeVector.Parse shouldn't be called")
 }
 
-func (t *TypeVector) StringOf(zv zcode.Bytes) string {
-	s := "vector["
-	comma := ""
+func (t *TypeVector) StringOf(zv zcode.Bytes, fmt OutFmt) string {
+	if len(zv) == 0 && fmt == OUT_FORMAT_ZEEK || fmt == OUT_FORMAT_ZEEK_ASCII {
```

Comment:
I think you want parentheses around the || expression.

Reply:
thanks for catching that (but my thorough tests would have caught it eventually 😮 )